src/scalecube-services/services/ServiceCall.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import { Router } from 'src/scalecube-services/services';
+
+export class ServiceCall{
+  router: Router;
+  constructor(router: Router, timeout:number = 5000){
+    this.router = router;
+  }
+  invoke(message:any) {
+    const inst = this.router.route(message);
+    if( inst ) {
+      return inst.service[message.method](...message.data);
+    }
```

Comment:
if service is not found on else statement need to throw error
service not found error.

Reply:
It gives a warnning
I can change it to error
```javascript
invoke(message:any) {
    const inst = this.router.route(message);
    if( inst ) {
      return inst.service[message.method](...message.data);
    }
    console.warn(`Service not found for message ${message}`);
  }
```